main/settings_celery.py

```diff
@@ -148,6 +148,12 @@
         "task": "learning_resources_search.tasks.update_featured_rank",
         "schedule": crontab(minute=30, hour=7),  # 3:30am EST
     },
+    "scrape-marketing-pages-every-1-hour": {
+        "task": "learning_resources.tasks.scrape_marketing_pages",
+        "schedule": get_int(
+            "SCRAPE_MARKETING_PAGES_SCHEDULE_SECONDS", 60 * 60 * 12
+        ),  # default is every hour
```

Comment:
\# default is every 12 hours

Reply:
fixed comment and also fixed name of scheduled task